docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -256,7 +181,10 @@ SELECT THETA_SKETCH_ESTIMATE(
 FROM ts_tutorial
 ```
 
-![Count distinct with Theta sketches and filters](../assets/tutorial-theta-08.png)
+The `APPROX_COUNT_DISTINCT_DS_THETA` function applies the following:
+
+* `DS_THETA`: Creates a new Theta sketch from the column of Theta sketches.
+* `THETA_SKETCH_ESTIMATE`: Calculates the distinct count estimate from the output of `DS_THETA` where the show is _Bridgerton_.
```

Comment:
>where the show is _Bridgerton_.

Shouldn't be in this description. The FILTER part doesn't belong to THETA_SKETCH_ESTIMATE